MAINTAINERS.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+The oneDPL project provides algorithms that are implemented using parallel patterns and bricks implemented
+for the various backends supported to allow oneDPL to run on CPU, GPU, and FPGA platforms. The API areas
+which a developer is a lead maintainer are listed below.
+
+
+| Name                  | Github ID           | API Area |
+| --------------------- | ------------------- | -------- |
+| Ruslan Arutyunyan     | @rarutyun           | PSTL Offload |
+| Konstantin Boyarinov  | @kboyarinov         | PSTL Offload |
+| Pavel Dyakov          | @paveldyakov        | Random number generator APIs |
+| Mikhail Dvorskiy      | @MikeDvorskiy       | Range-based algorithms, Sort algorithms |
+| Adam Fidel            | @adamfidel          | Scan algorithms, Experimental Scan KT |
+| Dan Hoeflinger        | @danhoeflinger      | Histogram, Transform, Iterators |
+| Alexandr Konovalov    | @Alexandr-konovalov | PSTL Offload |
+| Sergey Kopienko       | @SergeyKopienko     | Find algorithms, Experimental Sort KT, validation tests |
+| Alexey Kukanov        | @akukanov           | oneDPL architecture, Experimental Sort KT |
+| Matt Michel           | @mmichel11          | By-segment algorithms, Search algorithms |
+| Julian Miller         | @julianmi           | Reduce algorithms |
+| Dmitriy Sobolev       | @dmitriy-sobolev    | Experimental Sort KT |
```

Comment:
Let's replace KT with Kernel Template. This is how we mention it in the documentation.
Could you add Sort algorithms area?

Reply:
The updated areas doesn't mention KT